feature/metis/manage-conversations/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/manage_conversations_moduleTest.kt

```diff
@@ -0,0 +1,121 @@
+import kotlin.test.Test
+
+
+@RunWith(RobolectricTestRunner::class)
+@Category(UnitTest::class)
+internal class manage_conversations_moduleTest : BaseComposeTest() {
+
+    @get:Rule
+    val koinTestRule = KoinTestRule.create {
+        androidContext(InstrumentationRegistry.getInstrumentation().context)
+
+        modules(coreTestModules)
+        modules(manageConversationsModule, sharedConversationModule)
+    }
+
+    private val courseId = 1L
+    private val conversationId = 22L
+    private val testContext = context.applicationContext as Application
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `the manage_conversations_module initializes ConversationOverviewViewModel without errors`() {
+        composeTestRule.testViewModelInitialization<ConversationOverviewViewModel>() {
+            parametersOf(testContext, courseId)
+        }
+    }
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `the manage_conversations_module initializes CreatePersonalConversationViewModel without errors`() {
+        composeTestRule.testViewModelInitialization<CreatePersonalConversationViewModel>() {
+            parametersOf(courseId)
+        }
+    }
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `the manage_conversations_module initializes CreateChannelViewModel without errors`() {
+        composeTestRule.testViewModelInitialization<CreateChannelViewModel>() {
+            parametersOf(courseId)
+        }
+    }
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `the manage_conversations_module initializes BrowseChannelsViewModel without errors`() {
+        composeTestRule.testViewModelInitialization<BrowseChannelsViewModel>() {
+            parametersOf(courseId)
+        }
+    }
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `the manage_conversations_module initializes ConversationSettingsViewModel without errors`() {
+        composeTestRule.testViewModelInitialization<ConversationSettingsViewModel> {
+            parametersOf(courseId, conversationId)
+        }
+    }
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `the manage_conversations_module initializes ConversationMembersViewModel without errors`() {
+        composeTestRule.testViewModelInitialization<ConversationMembersViewModel> {
+            parametersOf(courseId, conversationId)
+        }
+    }
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `the manage_conversations_module initializes ConversationAddMembersViewModel without errors`() {
+        composeTestRule.testViewModelInitialization<ConversationAddMembersViewModel> {
+            parametersOf(courseId, conversationId)
+        }
+    }
+
+    /**
+     * Test if a ViewModel can be initialized without errors.
+     * @param parameters The parameters to pass to the ViewModel constructor
+     */
+    @OptIn(KoinInternalApi::class)
+    private inline fun <reified T: ViewModel> ComposeContentTestRule.testViewModelInitialization(
```

Comment:
We should probably move this method to core/common-test if possible to avoid code duplication.

Reply:
I tried moving it to BaseComposeTest, but there were errors, which is why I chose this solution.